docs/source/gbd2019_models/hemoglobin_anemia_and_iron_deficiency/anemia_impairment/index.rst

```diff
@@ -7,3 +7,173 @@ Anemia Impairment
+
+A complete list of the causes included in the causal attribution process for anemia include: 
+
+  P. falciparum parasitaemia without clinical malaria; P. vivax parasitaemia without clinical malaria; Clinical malaria; Schistosomiasis; Hookworm disease; Other neglected tropical diseases; Maternal haemorrhage; Vitamin A deficiency (under 15 years only); Other infectious diseases; Peptic ulcer disease; Gastritis; Stage III chronic kidney disease; Stage IV chronic kidney disease ; Stage V chronic kidney disease; End stage renal disease; Uterine fibroids; Menstrual disorders; Other haemoglobinopathies and haemolytic anaemias; Other endocrine, nutrition, blood, and immune disorders; G6PD deficiency; Hemizygous G6PD deficiency; Beta-thalassaemia major; Beta-thalassaemia trait; Haemoglobin E trait; Haemoglobin E/beta-thalassaemia; Haemoglobin H disease; Homozygous sickle cell and severe sickle cell/beta-thalassaemia parent; Haemoglobin SC disease; Mild sickle cell/beta-thalassaemia; Sickle cell trait; HIV; Cirrhosis and other chronic liver diseases, decompensated; Ulcerative colitis; Crohn’s disease; dietary iron deficiency; other infectious diseases; other neglected tropical diseases; other endocrine, nutrition, blood, and immune disorders; and other hemoglobinopathies and hemolytic anemias.
+
+The following table displays the cause- and sex-specific hemoglobin shifts used for the anemia causal attribution process in GBD 2019. These shifts are hosted `here <https://stash.ihme.washington.edu/projects/MNCH/repos/anemia_causal_attribution/browse/priors/hb_shifts.csv>`_.
+
+.. csv-table:: Cause- and Sex-Specific Hemoglobin Shifts
+  :widths: 15 15 15
+  :file: hb_shifts.csv
+
+Iron Responsive Causes
+^^^^^^^^^^^^^^^^^^^^^^
+
+A list of which causes of anemia are iron responsive can be found in the excel sheet hosted `here <https://stash.ihme.washington.edu/projects/MNCH/repos/anemia_causal_attribution/browse/in_out_meid_map.xlsx>`_.
+
+Vivarium Modeling Strategy
+--------------------------
+
+Scope
+++++++++
+
+The Vivarium modeling strategy for the anemia impairment will first rely on the :ref:`Hemoglobin Distribution Model <2019_hemoglobin_distribution>`. Vivarium simulants should first be assigned a hemoglobin value and then their anemia status can be evaluated by the hemoglobin thresholds shown on this page. YLDs should be accrued according to the severity-specific disability weights listed in this document. Specific causes (or groups of causes, for example iron-responsive causes) of anemia can also be assigned to individual if relevant/necessary for the project.
+
+Restrictions
+++++++++++++
+
+.. list-table:: GBD 2019 Anemia Impairment Restrictions
+   :widths: 15 15 20
+   :header-rows: 1
+
+   * - Restriction Type
+     - Value
+     - Notes
+   * - Male only
+     - False
+     -
+   * - Female only
+     - False
+     - Note the pregnancy adjustment for women of reproductive age
+   * - Age group start
+     - Birth (age_group_id=2)
```

Comment:
Should this be "Early Neonatal" instead of "Birth"?  I think "at birth" has its own age group id (164?), e.g. if there's birth prevalence for anemia.